collect_app/src/androidTest/java/org/odk/collect/android/support/pages/AddProjectDialogPage.kt

```diff
@@ -22,6 +22,11 @@ internal class AddProjectDialogPage(rule: ActivityTestRule<*>) : Page<AddProject
         return this
     }
 
+    fun inputProjectColor(projectColor: String?): AddProjectDialogPage {
```

Comment:
This should just be `String` rather than `String?` I think.

Reply:
Right, fixed.